_posts/2020-06-03-project-workflow

```diff
@@ -0,0 +1,87 @@
+---
```

Comment:
@ABizzinotto hey, this file should be `.md`

Reply:
Done